creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -853,8 +849,10 @@ class SnapbackSM {
           time: Date.now()
         })
       } catch (e) {
-        // If unable to initialize `endpointToSpIdMap`, disable reconfig
-        this.highestReconfigModeEnabled = RECONFIG_MODES.RECONFIG_DISABLED.key
+        // Disable reconfig after failed `updateEndpointToSpIDMap()` call
+        this.updateEnabledReconfigModesSet({
```

Comment:
just so i understand, this is basically a short circuit to not do any reconfig ops if the updateEndpointToSpIdMap function fails? and the next iteration it gets reset so it could proceed potentially?

Reply:
exactly yep, also this already existed in the PR, i just refactored it to what i felt was simpler/less error-prone